src/applications/personalization/dashboard-2/components/health-care/HealthCareCard.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import NotificationCTA from '../NotificationCTA';
+
+const HealthCareCard = ({ type, cardProperties, noActiveData }) => {
+  let cardTitle = cardProperties?.cardTitle;
+  let line1 = cardProperties?.line1;
+  let line2 = cardProperties?.line2;
+  let line3 = cardProperties?.line3;
+  let sectionTitle = cardProperties?.sectionTitle;
+
+  const CTA = {
+    text: cardProperties?.ctaText,
+    icon: cardProperties?.ctaIcon,
+    href: cardProperties?.ctaHref,
+    ariaLabel: cardProperties?.ctaAriaLabel,
+  };
+
+  if (type === 'messages') {
```

Comment:
so the idea is that once we have "real" components for messages and appointments, we can kill the `type` prop?

Reply:
That's right :)